test_infra/Cargo.toml

```diff
@@ -8,6 +8,7 @@ version = "0.1.0"
 dirs = "5.0.1"
 epoll = "4.3.3"
 libc = "0.2.153"
+nix = { version = "0.29.0", features = ["signal"] }
```

Comment:
I don't want to add a nix dependency.

Reply:
Why? Is there anything else to consider?

Reply:
It's large and unnecessary. What was the problem with `libc::kill` ?

Reply:
There is no problem. `nix::kill` called the `libc::kill`. `nix` wraps up errno for capturing errors.

Reply:
Please can you go back to using `libc::kill` directly.

Reply:
Okay. In fact, I have already fixed it.